### DISCUSSION TITLE
Why docto on windows detect as Trojan?

### DISCUSSION THREAD

**Discussion by D1maD1m0nd** [tobya/DocTo#227] [Q&A] [answered]
I installed docto on the server and tried to open it, but the antivirus complained about the Trojan, why?

**Comment by tobya** [comment, marked as answer]
Its a false positive. 

Which version did you install?

The fact that it uses COM objects to manipulate word documents is probably detected and given a false reading as a trojan.  However this is exactly what the app is supposed to do.

Thanks for using DocTo
